kubric/__init__.py

```diff
@@ -47,15 +47,9 @@
 from kubric.core.objects import Cube
 from kubric.core.objects import FileBasedObject
 
-from kubric.core.traits import Vector3D
-from kubric.core.traits import Scale
-from kubric.core.traits import Quaternion
-from kubric.core.traits import RGB
-from kubric.core.traits import RGBA
-from kubric.core.traits import AssetInstance
-
 from kubric.custom_types import AddAssetFunction
 from kubric.custom_types import PathLike
+from kubric.custom_types import Quaternion
```

Comment:
```suggestion
from pyquat import Quaternion
```

Reply:
The point of this is to make it clear what's the rotational type used within the library?